apps/blog/.env

```diff
@@ -1 +1,2 @@
-NEXT_PUBLIC_APEX_URL = "https://pancakeswap.finance"
\ No newline at end of file
+NEXT_PUBLIC_APEX_URL = "https://pancakeswap.finance"
+NEXT_PUBLIC_GTAG = "G-334KNG3DMQ"
```

Comment:
Let's set in vercel instead of here to avoid forks/local

Reply:
done